include/unifex/get_stop_token.hpp

```diff
@@ -16,15 +16,44 @@
 #pragma once
 
 #include <unifex/stop_token_concepts.hpp>
+#include <unifex/inplace_stop_token.hpp>
 #include <unifex/tag_invoke.hpp>
 #include <unifex/type_traits.hpp>
+#include <unifex/coroutine.hpp>
 #include <unifex/unstoppable_token.hpp>
 
+#if !UNIFEX_NO_COROUTINES
+#include <unifex/await_transform.hpp>
+#endif
+
 #include <unifex/detail/prologue.hpp>
 
 namespace unifex {
 namespace _get_stop_token {
   inline const struct _fn {
+  private:
+#if !UNIFEX_NO_COROUTINES
+    template <typename StopToken>
+    struct awaiter_ {
```

Comment:
I don't know how important the naming convention is, here, but "private" Unifex types seem to _start_ with underscores rather than end with them...